src/org/omegat/gui/editor/EditorController.java

```diff
@@ -1456,6 +1456,36 @@ public void nextTranslatedEntry() {
         nextTranslatedEntry(true);
     }
 
+    private void linkedEntry(boolean forward, String linked) {
+        iterateToEntry(forward, ste -> {
+                TMXEntry info = Core.getProject().getTranslationInfo(ste);
+                if (String.valueOf(info.linked).equals(linked)) {
+                    return true;
+                }
+                return false;
+            });
```

Comment:
This can be simplified because second argument is `@NotNull`

```suggestion
        iterateToEntry(forward, ste -> linked
            .equals(Core.getProject().getTranslationInfo(ste).linked));
```

Reply:
```suggestion
        iterateToEntry(forward, ste -> {
                TMXEntry info = Core.getProject().getTranslationInfo(ste);
                return String.valueOf(info.linked).equals(linked);
            });
```

this will be.